package.json

```diff
@@ -77,6 +77,7 @@
     "minimist": "1.2.x",
     "moment-timezone": "^0.5.23",
     "morgan": "^1.9.1",
+    "mountebank": "^1.16.0",
```

Comment:
This runs as a server and is started using the command `mb` in the selfservice directory